packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -85,7 +85,16 @@ export default class {
   }
 
   async getAllRemotes() {
-    return (await this._remotes.get()).map(_ => obfuscateRemote(_))
+    const remotes = this._remotes.get().map(async remote => {
```

Comment:
`get()` probably returns a promise, which does not have a `map` method.

You can use `asyncMap` here if you want.